doc/administrator/editing.rst

```diff
@@ -66,6 +76,8 @@ corresponding to this field is *Related Postgres table* in the admin interface.
     It is recommended to limit the editing to 30 editable layers per user.
 
 
+.. _administrator_editing_editable:
```

Comment:
Pretty sure you should move back this anchor before "Adding or making layers editable".

Reply:
You're right